src/OpenLoco/src/Windows/MapWindow.cpp

```diff
@@ -1508,11 +1510,92 @@ namespace OpenLoco::Ui::Windows::MapWindow
         }
     }
 
-    // 0x0046CED0
-    static void sub_46CED0()
+    // 0x00478265
+    static void sub_478265(int8_t* buffer)
```

Comment:
I'm not really a fan of marking this as signed. Its just that road elements have the highest bit set (and the end of the array is 0xFF). Its not really signed.

Reply:
In addition the other user of sub_478265 using uint8_t